cips/cip-14.md

```diff
@@ -28,39 +28,48 @@ By enabling ICA on an accelerated timeline, Celestia can enable battle-tested pr
 
+Adoption of the host module implies the addition of new parameters:
 
-### Definitions 
+| Module.Parameter      | Proposed value                                                                                                                                                                                                                                                                                                                                                                                                                                                        | Summary                                                                                                                                                                                                        | Changeable via Governance |
+|-----------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------------------|
+| icahost.HostEnabled   | `true`                                                                                                                                                                                                                                                                                                                                                                                                                                                                | Controls a chains ability to service ICS-27 host specific logic.                                                                                                                                               | False                     |
+| icahost.AllowMessages | `["/ibc.applications.transfer.v1.MsgTransfer", "/cosmos.bank.v1beta1.MsgSend", "/cosmos.staking.v1beta1.MsgDelegate", "/cosmos.staking.v1beta1.MsgBeginRedelegate", "/cosmos.staking.v1beta1.MsgUndelegate", "/cosmos.staking.v1beta1.MsgCancelUnbondingDelegation", "/cosmos.distribution.v1beta1.MsgSetWithdrawAddress", "/cosmos.distribution.v1beta1.MsgWithdrawDelegatorReward", "/cosmos.distribution.v1beta1.MsgFundCommunityPool", "/cosmos.gov.v1.MsgVote"]` | Provides the ability for a chain to limit the types of messages or transactions that hosted interchain accounts are authorized to execute by defining an allowlist using the Protobuf message type URL format. | False                     |
```

Comment:
Thanks for the table!

> False

Should this be False? Potentially there may be desire to add more in the future?

Reply:
I expect more messages to be added to this list in the future but IMO this list should be expanded via the CIP process rather than on-chain governance.